web/index.css

```diff
@@ -416,3 +380,31 @@ div.toggle {
   padding: 4px;
   background: #3f3f3f;
 }
+
+.sharing-dialog {
+  top: 100px;
+  left: 100px;
+  right: 100px;
+}
```

Comment:
On your screenshot it doesn't look that, it depends on your screen I guess. On my big widescreen, it looks extremly wide.